src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -295,6 +297,11 @@ private ICitizenData spawnCitizenOnPosition(
         entity.setPosition(spawnPoint.getX() + HALF_BLOCK, spawnPoint.getY() + SLIGHTLY_UP, spawnPoint.getZ() + HALF_BLOCK);
         world.addEntity(entity);
 
+        AbstractCitizenEvent spawnEvent = citizenData.isChild() ? new CitizenBornEvent() : new CitizenSpawnedEvent();
+        spawnEvent.setEventPos(spawnPoint);
+        spawnEvent.setCitizenName(citizenData.getName());
+        colony.getEventManager().addEventDescription(spawnEvent);
```

Comment:
I feel like those things would fit better into the constructor of the event instead of manually setting pos and name.

Reply:
thats a relic from a changed implementation, i thought i changed it, but seems like i didn't.

Reply:
This time i really change that, and not just planned to do it xD

Reply:
:+1: xD